rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -147,4 +152,15 @@ private String getLoggedInUserPreferredName() {
         final AllSharedPreferences allSharedPreference = RDTApplication.getInstance().getContext().allSharedPreferences();
         return allSharedPreference.getANMPreferredName(allSharedPreference.fetchRegisteredANM());
     }
+
+    private JSONArray getLocations() {
+        JSONArray jsonArray = new JSONArray();
+        AllSharedPreferences allSharedPreferences = RDTApplication.getInstance().getContext().allSharedPreferences();
+        String defaultLocationUuid = allSharedPreferences.fetchDefaultLocalityId(allSharedPreferences.fetchRegisteredANM());
+        List<String> locations = LocationHelper.getInstance().getOpenMrsLocationHierarchy(defaultLocationUuid, false);
+        for (String location : locations) {
+            jsonArray.put(location);
+        }
+        return jsonArray;
+    }
```

Comment:
Is this logic verified? It looks like it pulls the entire location hierarchy and not the sibling locations as described here https://github.com/onaio/rdt-standard/issues/349#issuecomment-700727776.

Reply:
I have checked this method it is returning only one location Indonesia. I have also tried other child-related methods but all of them returning null.

Reply:
Actually, I can't determine whats the exact method will be used for that. Most of the method which I have tried that is returning null or not record.

Reply:
I changed the location hierarchy to have a few more levels. Could you verify that more locations show up in the hierarchy?

The user `indtester1` should be assigned to `Indonesia Location 3`. The dropdown should show `Indonesia Location 1, 2 and 3`.